src/Microsoft.AspNet.Mvc.Razor/RazorPage.cs

```diff
@@ -546,8 +546,8 @@ public Task<HtmlString> RenderSectionAsync([NotNull] string name)
         /// </summary>
         /// <returns>A<see cref="Task{HtmlString}"/> that represents the asynchronous flush operation and on
         /// completion returns a <see cref="HtmlString.Empty"/>.</returns>
-        /// <remarks>The value returned is a token value that allows FlushAsync to succeed. However the
-        /// value does not represent the rendered content.</remarks>
+        /// <remarks>The value returned is a token value that allows FlushAsync to  to work directly in the
+        /// an HTML section. However the value does not represent the rendered content.</remarks>
```

Comment:
Could you please fix the comment to say - "The value returned is a token value that allows FlushAsync to work directly in an HTML section."

Currently you have 2 "to"s in the sentence and "the" and "an" before HTML section.

Also, have you signed a [CLA](https://github.com/aspnet/Home/blob/master/CONTRIBUTING.md#contributing-code-and-content) as yet? cc @Eilon